app/src/main/java/com/sdp/movemeet/chat/ChatActivity.java

```diff
@@ -107,6 +129,64 @@ protected void onCreate(Bundle savedInstanceState) {
                 new MyScrollToBottomObserver(messageRecyclerView, mFirebaseAdapter, mLinearLayoutManager)
         );
 
+        drawerLayout=findViewById(R.id.drawer_layout);
+        navigationView=findViewById(R.id.nav_view);
+        textView=findViewById(R.id.textView);
+        toolbar=findViewById(R.id.toolbar);
+
+        navigationView.bringToFront();
+        ActionBarDrawerToggle toggle=new
+                ActionBarDrawerToggle(this,drawerLayout,toolbar,R.string.navigation_drawer_open,R.string.navigation_drawer_close);
+        drawerLayout.addDrawerListener(toggle);
+
+        View hView =  navigationView.inflateHeaderView(R.layout.header);
+
+        fullName = hView.findViewById(R.id.text_view_profile_name);
+        phone = hView.findViewById(R.id.text_view_profile_phone);
+        email = hView.findViewById(R.id.text_view_profile_email);
+
+        toggle.syncState();
+        navigationView.setNavigationItemSelectedListener(this::onNavigationItemSelected);
+        navigationView.setCheckedItem(R.id.nav_home);
+
+        handleRegisterUser();
```

Comment:
Why did you add this function here ("handleRegisterUser")? Why do you need those information about the user here? (Because personally I only needed the user's fullName for the chat...)

Reply:
Ok fine, I see! Thank you!